### ISSUE TITLE
Review Ticket Week 4

### ISSUE THREAD

**Issue by benjaminlee24** [benjaminlee24/gang-ds-blog#6]
[My Video](https://www.wevideo.com/view/2801525847)

**Comment by NicolasMosqueda** [comment]
### Thoughts by Nicolas Mosqueda
Your video was clear and showed that you completed the assignment. I do see a lot of potential in your video-making skills and can see you making great videos down the line. Your reflection on the video was very well written and insightful
### Grade 2.7

**Comment by jm1021** [comment]
I asked not video from camera.    It is required to learn how to do this from machine.  Not sure you shared all the technicals.  2.5

**Comment by clairehzhao** [comment]
2.7
- voice over is clear
- used wevidoe nicely

**Comment by SRIHITAKOTT1213** [comment]
2.7 some changes need to be made on the screen next time :)

**Comment by mmaxwu** [comment]
2.7
Screen recording would've made it clearer

**Comment by nsk1207** [comment]
2.7 screen recording would have made your video clearer but still good overall

**Comment by gwang1224** [comment]
2.7, A little hard to follow, completed all requirements.

**Comment by Emaad-Mir** [comment]
Grade: 2.7
- did a good job at demonstrating functionality
- could have used screen recording to show his process

**Comment by EdwinKuttappi** [comment]
2.7, your voice was clear and you seemed to know what you were doing

**Comment by T-Dev-CCM** [comment]
2.7. You properly demonstrated what you were told to and your voiceover helped

**Comment by amitha-sanka** [comment]
2.7

- clear voiceover 
- used wevideo 
- included all parts

**Comment by realethantran** [comment]
2.7
Good job you had each local server running, but next time it may be better to add some text on the screen so that the video is easier to understand.

**Comment by rohinsood** [comment]
> 2.7+

### Notes
 - Well done with your organization and coherence! Perhaps add some text or visual clues on your screen for greater immersion.
